### PR TITLE
Add blackbox_exporter docs. Update changelog.

### PR DESCRIPTION
**Pull request by keithf4**
# Description  

Add documentation for the blackbox_exporter

## Type of change  
Please check all options that are relevant  
- [ ] Bug fix (change which fixes an issue)  
- [ ] Feature (change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [x] Documentation update only  

# How Has This Been Tested?  

**Tested Configuration**:  
- [ ] CentOS, Specify version(s):  
- [ ] PostgreQL, Specify version(s):  
- [x] docs tested with hugo <0.60  

Tested with playbook(s):  
# Checklist:  
- I have made corresponding changes to:  
    - [x] the documentation  
    - [ ] the release notes  
    - [ ] the upgrade doc

### CODE REVIEW

File: hugo/content/changelog/_index.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ weight: 5
 
 ### New Features
 
+ * blackbox_exporter is now available for blackbox probing of endpoints over HTTP, HTTPS, DNS, TCP and ICMP. More info about this exporter is available via their github repo: https://github.com/prometheus/blackbox_exporter
```
Comment (hunleyd) [inline hugo/content/changelog/_index.md:10]:
```suggestion
 * blackbox_exporter is now available for Blackbox probing of endpoints over HTTP, HTTPS, DNS, TCP and ICMP. More info about this exporter is available via their github repo: https://github.com/prometheus/blackbox_exporter
```

Comment (hunleyd) [reply hugo/content/changelog/_index.md:10]: I know this is lowercase on their own site but it's a name, right?

File: hugo/content/exporter/_index.md [hunk 2]
```diff
@@ -94,6 +98,15 @@ The following pgMonitor configuration files should be placed according to the fo
 | postgres/queries_*.yml | `/etc/postgres_exporter/##/queries_*.yml`  |
 | postgres/pgbackrest-info.sh | `/usr/bin/pgbackrest-info.sh` |
 
+##### blackbox_exporter
+
+The following pgMonitor configuration files should be placed according to the following mapping:
+
+| pgMonitor Configuration File | System Location |
+|------------------------------|-----------------|
+| blackbox/blackbox_exporter.sysconfig  | `/etc/sysconfig/blackbox_exporter`   |
```
Comment (hunleyd) [inline hugo/content/exporter/_index.md:107]:
```suggestion
| blackbox/blackbox_exporter.sysconfig  | {{ shell }}/etc/sysconfig/blackbox_exporter{{ /shell }}   |
```

File: hugo/content/exporter/_index.md [hunk 3]
```diff
@@ -94,6 +98,15 @@ The following pgMonitor configuration files should be placed according to the fo
...
+| blackbox/crunchy-blackbox.yml| `/etc/blackbox_exporter/crunchy-blackbox.yml` |
```
Comment (hunleyd) [inline hugo/content/exporter/_index.md:108]:
```suggestion
| blackbox/crunchy-blackbox.yml| {{ shell }}/etc/blackbox_exporter/crunchy-blackbox.yml{{ /shell }} |
```

File: hugo/content/exporter/_index.md [hunk 4]
```diff
@@ -232,6 +245,10 @@ The `/etc/postgres_exporter/##/crontab.txt` file is meant to be a guide for how
 
 The script requires being run by a database superuser by default since it must be able to run a scan on every table. If you'd like to not run it as a superuser, you will have to create a new role that has read permissions on all tables in all schemas that are to be monitored for bloat. You can then change the user in the connection string option to the script.
 
+##### Blackbox Exporter
+
+The configuration file for the blackbox_exporter provided by pgMonitor (`/etc/blackbox_exporter/crunchy-blackbox.yml`) provides a probe for monitoring any ivp4 tcp port status. The actual target and port being monitored is controlled via the Prometheus target configuration system. Please see the pgMonitor Prometheus documentation for further details. If any additional blackbox probes are desired, please see the upstream documentation.
```
Comment (hunleyd) [inline hugo/content/exporter/_index.md:250]:
```suggestion
The configuration file for the blackbox_exporter provided by pgMonitor ({{ shell }}/etc/blackbox_exporter/crunchy-blackbox.yml{{ /shell }}) provides a probe for monitoring any IPv4 TCP port status. The actual target and port being monitored are controlled via the Prometheus target configuration system. Please see the pgMonitor Prometheus documentation for further details. If any additional Blackbox probes are desired, please see the upstream documentation.
```

File: hugo/content/prometheus/_index.md [hunk 5]
```diff
@@ -133,6 +133,14 @@ The below files dictate how Prometheus and Alertmanager will behave at runtime f
...
 | `/etc/prometheus/auto.d/*.yml` | You will need at least one file with a final `.yml` extension. Copy the example files to create as many additional targets as needed.  Ensure the configuration files you want to use do not end in `.yml.example` but only with `.yml`. Note that in order to use the provided Grafana dashboards, the extra "exp_type" label must be applied to the target and be set appropriately (pg or node). See the example target files provided for how to set the labels for postgres or node exporter targets. |
 
+#### Blackbox Exporter
+
+By default, the blackbox exporter probes are commented out in the `crunchy-prometheus.yml` file. Please see the notes in that commented out section. For the default ipv4 tcp port targets that pgMonitor configures the blackbox_exporter with, the desired monitoring targets can be configured under the `static_configs: targets` section of the `blackbox_tcp_services` job. Some examples for grafana & patroni are given there. It is also possible to create another auto scrape target directory similar to `auto.d` and manage your blackbox targets more dynamically.
```
Comment (hunleyd) [inline hugo/content/prometheus/_index.md:138]:
```suggestion
By default, the Blackbox exporter probes are commented out in the {{ shell }}crunchy-prometheus.yml{{ /shell }} file; please see the notes in that commented out section. For the default IPv4 TCP port targets that pgMonitor configures the blackbox_exporter with, the desired monitoring targets can be configured under the {{ yaml }}static_configs: targets{{ /yaml }} section of the {{ yaml }}blackbox_tcp_services{{ /yaml }} job; some examples for Grafana & Patroni are given there. It is also possible to create another auto-scrape target directory similar to {{ shell }}auto.d{{ /shell }} and manage your blackbox targets more dynamically.
```

File: hugo/content/prometheus/_index.md [hunk 6]
```diff
@@ -133,6 +133,14 @@ The below files dictate how Prometheus and Alertmanager will behave at runtime f
...
+
+If you configure additional probes besides the one that pgMonitor comes with, you will need to create a different prometheus job_name for them for the given `params: module` name.
```
Comment (hunleyd) [inline hugo/content/prometheus/_index.md:140]:
```suggestion
If you configure additional probes beyond the one that pgMonitor comes with, you will need to create a different Prometheus {{ yaml }}job_name{{ /yaml }} for them for the given {{ yaml }}params: module{{ /yaml }} name.
```

File: hugo/content/prometheus/_index.md [hunk 7]
```diff
@@ -133,6 +133,14 @@ The below files dictate how Prometheus and Alertmanager will behave at runtime f
...
+
+An example rules file for monitoring blackbox probes is available in the alert-rules.d folder (`crunchy-alert-rules-blackbox.yml.example`).
```
Comment (hunleyd) [inline hugo/content/prometheus/_index.md:142]:
```suggestion
An example rules file for monitoring Blackbox probes, {{ shell }}crunchy-alert-rules-blackbox.yml.example{{ /shell }}, is available in the {{ shell }}alert-rules.d{{ /shell }} folder.
```

File: hugo/content/changelog/_index.md (hunk 1 above)
Comment (keithf4) [reply hugo/content/changelog/_index.md:10]: If it's referring specifically to the name of the product and not using the `blackbox_exporter` name, then yes, I'd agree capitalizing Blackbox would be a good idea. However, in this case, it's referring to the generalized description of the word "blackbox" meaning it's probing something without knowing what its real intention is other than getting a tcp response.

File: hugo/content/exporter/_index.md (hunk 4 above)
Comment (keithf4) [reply hugo/content/exporter/_index.md:250]: Agree here, this can be made capital

File: hugo/content/exporter/_index.md (hunk 2 above)
Comment (keithf4) [reply hugo/content/exporter/_index.md]: Actually, I don't think these submodule changes have been applied to the pgmonitor repository. I made sure it was all up to date, but when I run hugo, these formatting flags aren't being applied

**Pull request merged**